srtcore/buffer_rcv.cpp

```diff
@@ -968,11 +968,17 @@ int CRcvBuffer::scanNotInOrderMessageLeft(const int startPos, int msgNo) const
     return -1;
 }
 
+#if SRT_DEBUG_TRACE_DRIFT
+bool CRcvBuffer::addRcvTsbPdDriftSample(uint32_t usTimestamp, const time_point& tsPktArrival, int usRTTSample, int msRcvBuf)
+{
+    return m_tsbpd.addDriftSample(usTimestamp, tsPktArrival, usRTTSample, msRcvBuf);
+}
+#else
```

Comment:
I suppose you don't need `msRcvBuf` as an argument, because the receiver buffer already knows this value (`getTimespan_ms()`).

Reply:
I guess the buffer fullness would indeed be helpful in the drift trace.
I just don't feel good that the debug tracing logic touches so many places. That might reduce the overall readability of the code. 🤔

Reply:
Ok. I'll have a think - if I can see a less intrusive way of getting that info into the log, I'll update the PR.

Reply:
Might be better to just attach those changes to issue #2477 in the form of a git diff patch.
This way it can be preserved. But I don't feel this PR should go to master.

Reply:
> Ok. I'll have a think - if I can see a less intrusive way of getting that info into the log, I'll update the PR.

Thank you, @oviano. And thanks for sharing your changes!